zallet/src/commands/migrate_zcash_conf.rs

```diff
@@ -507,7 +507,7 @@ fn build_actions() -> HashMap<&'static str, Action> {
         .chain(Action::map_multi(
             "nuparams",
             |config| &mut config.regtest_nuparams,
-            |value| RegTestNuParam::try_from(value).map_err(|_| ()),
+            |value| RegTestNuParam::try_from(value.to_string()).map_err(|_| ()),
```

Comment:
The reason I was confused is that the code altered here was working perfectly. I've just written additional testing and reconfirmed that the existing code works fine for serializing a `ZalletConfig`. The fact that serialization works fine but deserialization apparently fails at runtime is an annoying serde aspect.